portals/knowledge-management/src/Pages/ViewArticle.vue

```diff
@@ -112,9 +116,15 @@ function getData() {
                         <div class="flex flex-col gap-3">
                             <div class="prose max-w-none">
                                 <h1>{{ article.name }}</h1>
-                                <div class="text-gray-500 flex items-center space-x-1 mb-4">
-                                    <ClockIcon class="h-4 w-4 flex-shrink-0" aria-hidden="true" />
-                                    <span class="text-xs">Last updated: {{ article.lastUpdated }}</span>
+                                <div class="flex mb-4">
+                                  <div v-if="portalViewCountFlag" class="text-gray-500 flex items-center space-x-1 mr-2">
+                                      <EyeIcon class="h-4 w-4 flex-shrink-0" aria-hidden="true" />
```

Comment:
This does not seem to be rendering for me on the frontend? Is it for you?

Reply:
@Orrison You ran the migrations?

Reply:
@jayushi-canyon I did. To be clear I mean I just don't see the Eye icon rendered on the screen. I think you forgot to import it at the top of the file like you did in AidingApp